### ISSUE TITLE
[Bug] restore page UI is still kinda strange

### ISSUE THREAD

**Issue by richardshiue** [AppFlowy-IO/AppFlowy#3462]
### Bug Description

Title says all

### How to Reproduce

Hold your mouse on one of the restore or delete permanently buttons after deleting the current page

### Expected Behavior

a more acceptable color

### Operating System

macOS

### AppFlowy Version(s)

0.3.2

### Screenshots

<img width="1060" alt="Screenshot 2023-09-20 at 12 07 02 PM" src="https://github.com/AppFlowy-IO/AppFlowy/assets/71320345/764f484f-a61b-4b1e-b831-87e3b56d084f">

**Comment by RawatDevanshu** [comment]
Hello, can i work on this?

**Comment by RawatDevanshu** [comment]
Hello, I have changed colors, I used colorScheme.surfaceVariant in place of .error in the banner background, and buttons used .tertiary and for outline .tertairyContainer. For highlights I used .onErrorContainer in place of .error respectively this is how it looks.


https://pasteboard.co/BDMKBho5JbZI.gif   --Dark
https://pasteboard.co/lkN9lQ0Zk8iH.gif      --Light

do these look just fine? 
it looks ugly like above when I switch to lavender or dandelion. But it could be dealt
